api/src/controllers/contract.js

```diff
@@ -149,7 +154,8 @@ router.post("/", passport.authenticate(["referent"], { session: false }), async
             .replace(/{{cta}}/g, `${APP_URL}/validate-contract?token=${recipient.token}&contract=${contract._id}`);
           const subject = `(RE)Valider le contrat d'engagement de ${contract.youngFirstName} ${contract.youngLastName} sur la mission ${contract.missionName} suite à modifications effectuées`;
           const to = { name: recipient.name, email: recipient.email };
-          await sendEmail(to, subject, htmlContent);
+          const cc = { name: recipient.youngName, email: recipient.cc };
+          await sendEmail(to, subject, htmlContent, { cc: cc ? cc : '' });
```

Comment:
`cc` is always truthy, it will send a mail in CC all the time event when it's not required. 

Untested suggestion:

```suggestion
          if (recipient.cc) {
            await sendEmail(to, subject, htmlContent, { cc: [{ name: recipient.youngName, email: recipient.cc }] });
          } else {
             await sendEmail(to, subject, htmlContent);
          }
```

(also I used an array see comment below)

Reply:
It works !